pre/remap_restart/tests/f522Toc360.yaml

```diff
@@ -45,4 +52,4 @@ output:
 slurm:
   account: g0610
```

Comment:
The yaml files of the five test cases all have an NCCS account (sponsor code) hardwired, although the account number differs between yaml files.  Is the account entry needed in the yaml files for the test cases?  Doesn't the test_remap_restarts.py script determine the account number?

Reply:
set to blank